cli/lib/stats.ts

```diff
@@ -0,0 +1,92 @@
+import { isSameDay, isSameMonth, isSameWeek } from 'date-fns';
+import fetch from 'node-fetch';
+import { readAppdata, saveAppdata } from './appdata';
+
+export enum StatsGroup {
+	STUDIO_CLI_USAGE_UNIQUE = 'studio-cli-usage-unique',
+}
+
+export enum StatsMetric {
+	SUCCESS = 'success',
+	FAILURE = 'failure',
+}
```

Comment:
Same thing here about reusing `StatsMetric` from `src/lib/bump-stats/types.ts` to clarify the expected structure of the `lastBumpStats` object.